lib/Sema/TypeCheckDeclObjC.cpp

```diff
@@ -928,7 +927,7 @@ bool swift::isRepresentableInObjC(const SubscriptDecl *SD, ObjCReason Reason) {
   if (!IndexResult)
     TypeRange = SD->getIndices()->getSourceRange();
   else
-    TypeRange = SD->getElementTypeLoc().getSourceRange();
+    TypeRange = SD->getElementTypeRepr()->getSourceRange();
```

Comment:
This part is technically a behavior change that opens us up to a crash-on-invalid case: `TypeLoc` does a null check and converts that to an empty `SourceRange`.

Reply:
Yeah, thank you. These should be `getElementTypeRepr()`.